config/webpack.config.js

```diff
@@ -36,7 +36,7 @@ const supportedLocales = () => {
 
```

Comment:
Does this file change any logic? If it's only style changes I bealive it should be done in a different PR

Reply:
didn't notice the auto format on save, removing the changes :)